Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>Instead could use: Number of new users signing up to your website.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Could you clarify this? I could be totally wrong, but I thought Neg binomial is the number of (failed+n-1) signups until N successful signups</p><p> </p><p>So if prob_signup = .2, and we want 3 people to signup, the Neg Binom would tell use how trials until 3 people sign up successfully?</p>

Reply:
<p>I find Thomas' example much easier when using the Gamma-Poisson (sorry for being obsessed) 😅 </p><p>Number of new users signing up to your website is Poisson distributed (number of successes without clear upper bound), but there is overdispersion , so you need more flexibility than a Poisson would allow --&gt; Gamma-Poisson</p><p> </p>

Reply:
<p>For thomas's example the gamma poisson thought process makes much more sense agreed.</p>

Reply:
<p>While that's true, if we don't even mention the Poisson, it doesn't really matter, it's just another name. These type of connections are really neat and rewarding for us but I don't think a newcomer will really appreciate them, for them, these are all just names and telling them "in PyMC and most other places it's called NegBinomial but we prefer Gamma-Poisson because of these theoretical considerations" does not add too much intuition - I think that's more advanced.</p>

Reply:
<p>I would use the example to number of sales made on a single day.</p>

Reply:
<p>That example wont fit well with the n,p parameterization. Are you also suggesting I change the parametrizaiton to mu gamma?</p><p> </p>